concat-js.php

```diff
@@ -232,13 +232,36 @@ function do_items( $handles = false, $group = false ) {
 						echo $inline_before;
 					}
 				}
+
 				if ( isset( $href ) ) {
+					$handles = implode( ',', $js_array['handles'] );
+
+					$load_mode = "defer";
+
+					// Stuff to skip loading JS async/defer
```

Comment:
Stupid atm, but don't want the load mode to actually break anything. The excluded list can be an option.